other-docs/guides/upgrading/v4.md

```diff
@@ -0,0 +1,98 @@
+
+
+### Block Editor
+
+The updated Block Editor in Altis v4 includes many changes, including deprecations to certain APIs. These changes may require updates to components such as sidebars or custom blocks, and these should be tested thoroughly. The [field guide for WordPress 5.4](https://make.wordpress.org/core/2020/03/03/wordpress-5-4-field-guide/) covers some of these changes.
+
+The Block Editor also includes [changes to markup generated by the block editor](https://make.wordpress.org/core/2020/03/02/markup-and-style-related-changes/). These changes will likely require updates to theme CSS to accomodate the new DOM structure, and may require changes to both JavaScript and PHP code that interacts with the structure of any relevant blocks.
+
+
+## Headline Features
+
+### Experience Blocks
+
+Altis v4 includes [Experience Blocks](https://www.altis-dxp.com/experience-blocks/), allowing content creators to seamlessly create personalised experiences.
+
+Experience Blocks is built on top of, and requires, the [Native Analytics](docs://analytics/native/) feature in the Analytics module. As Native Analytics gathers analytical data about end users, care should be taken to ensure you are meeting appropriate privacy and consent laws around personal data. Provided Native Analytics is enabled, no further work is required to enable or integrate Experience Blocks.
+
+Alongside Experience Blocks, some analytics data has changed to improve the user experience. Underlying support for analytics data has been enhanced to support multiple values for each field in metrics and attributes.
+
+Out of the box, the Audiences builder includes many useful fields from events, allowing users to build their audiences using live data. Additional data can be added to this UI using the new [`register_field()`](docs/native/server-side-api.md#audience-functions) function, ensuring that the most relevant fields are displayed to your users.
+
+
+### WordPress 5.4 "Adderley"
+
+The latest CMS module brings with it WordPress version 5.4. Some of the highlights are:
+
+- New [Social Icons and Buttons blocks](https://make.wordpress.org/core/2020/02/27/new-or-updated-blocks-in-wordpress-5-4/)
+- TikTok embeds in the block editor
+
+You can [find the full WordPress 5.4 release notes here](https://wordpress.org/news/2020/03/adderley/), and the developer [field guide here](https://make.wordpress.org/core/2020/03/03/wordpress-5-4-field-guide/).
+
+Note that in line with our development and user experience philosophies, some changes in the CMS module may differ from WordPress changes. Notably in Altis v4, the [fullscreen editor mode](https://make.wordpress.org/core/2020/03/03/fullscreen-mode-enabled-by-default-in-the-editor/) is not enabled by default.
+
+
+## Other Developer Features and APIs
```

Comment:
The security module now has PHP Basic Auth and limit login attempts features.

Reply:
Search now has the advanced mode, strict/loose matching option and better multi language support.